### PR TITLE
mcmb: Remove unused and debugging functionality

### PR DESCRIPTION
**Pull request by johnsonjh**
* mcmb: Remove unused and debugging functionality
  * Also output error messages in native locale language

### CODE REVIEW

**Review by johnsonjh — COMMENTED** [review]
👍🏻

File: utils/mcmb/mcmb.c [hunk 1]
```diff
@@ -409,23 +392,20 @@ static struct cmb_xitem *cmb_transform_find;
     for (n = 1; n < nitems; n++)                                              \
       {                                                                       \
         (void)memcpy(&xitem, &items[n], sizeof ( struct cmb_xitem * ));       \
-        if (!opt_silent && !opt_quiet)                                        \
+        if (!opt_quiet)                                                       \
```
Comment (aremmell) [inline utils/mcmb/mcmb.c:395]: trailing whitespace?

**Review by aremmell — CHANGES_REQUESTED** [review]
i think you left whitespace there that you didn't mean to

File: utils/mcmb/mcmb.c (hunk 1 above)
Comment (johnsonjh) [reply utils/mcmb/mcmb.c:395]: It's OK—it's inside a macro where the continuation backslash is right-justified.

**Review by aremmell — APPROVED** [review]

**Pull request merged**